backend/src/connector/dto.rs

```diff
@@ -0,0 +1,1209 @@
+/*
+* Note: This file is mostly autogenerated
```

Comment:
What parts was changed after the file was generated? Is it possible to move the changes into separate file (for example if the new functions was added, then it is better to create a separate file and add them there)?